recipes/log.c/all/conanfile.py

```diff
@@ -0,0 +1,63 @@
+from conans import ConanFile, CMake, tools
+import glob
+from conans.errors import InvalidNameException
+
+class logcConan(ConanFile):
+    name = "log.c"
+    version = "0.1.0"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rxi/log.c"
+    description = "A simple logging library implemented in C99"
+    topics = ("logging", "log", "logging-library", "conan", "logc","purec","c99")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "log_use_color": [True, False] }
```

Comment:
This opion should not be namespaced with `log_`.
I would rename it to a simple `color`.